examples/pico-dsl-extension/assets/pico-language-configuration.json

```diff
@@ -0,0 +1,18 @@
+{
+  "comments": {
+    "lineComment": ";",
+    "blockComment": []
+  },
+  "autoClosingPairs": [
+    { "open": "{", "close": "}" },
```

Comment:
these brackets do not occur in Pico

Reply:
ah, I copied them from a previous project :)